packages/core/src/components/AvatarGroup/__stories__/AvatarGroup.stories.tsx

```diff
@@ -16,6 +17,8 @@ import TableBody from "../../Table/TableBody/TableBody";
 import TableRow from "../../Table/TableRow/TableRow";
 import TableCell from "../../Table/TableCell/TableCell";
 import styles from "./AvatarGroup.stories.module.scss";
+import type { Meta, StoryObj } from "@storybook/react";
+import type { CounterColor } from "src/components/Counter/CounterConstants";
```

Comment:
Let's use relative paths since we currently don't have a config for absolute paths yet